### PR TITLE
docs(http_scrape source): Fix title and alias

### PR DESCRIPTION
**Pull request by spencergilbert**
https://deploy-preview-14517--vector-project.netlify.app/docs/reference/configuration/sources/http_scrape

### CODE REVIEW

File: website/cue/reference/components/sources/http_scrape.cue [hunk 1]
```diff
@@ -2,7 +2,6 @@ package metadata
 
 components: sources: http_scrape: {
 	title: "HTTP Scrape"
-	alias: "http_scrape"
```
Comment (spencergilbert) [inline website/cue/reference/components/sources/http_scrape.cue:5]: @neuronull `alias` here is used for renamed components 👍

Comment (neuronull) [reply website/cue/reference/components/sources/http_scrape.cue:5]: Good to know thanks :+1:

**Review by neuronull — APPROVED** [review]

File: website/cue/reference/components/sources/http_scrape.cue (hunk 1 above)
Comment (spencergilbert) [reply website/cue/reference/components/sources/http_scrape.cue:5]: I didn't realize this on the initial PR either, I assume you were going for the `HTTP Scrape` => `http_scrape` mapping?

Comment (neuronull) [reply website/cue/reference/components/sources/http_scrape.cue:5]: I think I was modelling it after `prometheus_scrape` cue files, which has `alias` set.

**Pull request merged**